projects/1.16/assets/xaeros-world-map/xaeroworldmap/lang/zh_cn.json

```
@@ -1,57 +1,120 @@
 {
+  "gui.xaero_open_map": "打开世界地图",
+  "gui.xaero_open_settings": "打开设置",
+  "gui.xaero_debug": "调试",
+  "gui.xaero_on": "开",
+  "gui.xaero_off": "关",
+  "gui.xaero_lighting": "灯光",
+  "gui.xaero_block_colours": "方块颜色",
+  "gui.xaero_accurate": "准确",
+  "gui.xaero_vanilla": "普通",
+  "gui.xaero_back": "返回",
+  "gui.xaero_load_chunks": "加载新区块",
+  "gui.xaero_update_chunks": "更新区块",
+  "gui.xaero_terrain_depth": "地形深度",
+  "gui.xaero_terrain_slopes": "地形坡度",
+  "gui.xaero_footsteps": "脚印",
+  "gui.xaero_light_levels": "光照等级",
+  "gui.xaero_requires_reload": "需要重新加载世界才能生效！更低=更好的性能 ",
+  "gui.xaero_flowers": "加载鲜花",
+  "gui.xaero_texture_compression": "材质压缩",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "gui.xaero_texture_compression": "纹理压缩",
```